src/content/docs/en/tutorial/5-astro-api/3.mdx

```diff
@@ -34,7 +34,7 @@ But, since you already have the directory `/tags/`, you can take advantage of an
 
 1. Create a new file `index.astro` in the directory `src/pages/tags/`.
 
-2. Navigate to `localhost:3000/tags` and verify that your site now contains a page at this URL. It will be empty, but it will exist.
+2. Navigate to `localhost:4321/tags` and verify that your site now contains a page at this URL. It will be empty, but it will exist.
```

Comment:
```suggestion
2. Navigate to `http://localhost:4321/tags` and verify that your site now contains a page at this URL. It will be empty, but it will exist.
```